docs/topics/modules/e2e/commit_push_changes_git.adoc

```diff
@@ -18,7 +18,19 @@ image::commit_menu.png[Git Commit Menu]
 .. Add a commit message describing your changes.
 .. Click btn:[Commit].
 +
-image::git_commit_dialog.png[Git Commit Dialog]
+//for hello world
```

Comment:
Suggestion for readability: put empty lines between each new set of such conditions to make reading the code easier.